src/main/kotlin/org/jetbrains/intellij/platform/gradle/tasks/aware/SplitModeAware.kt

```diff
@@ -0,0 +1,37 @@
+// Copyright 2000-2024 JetBrains s.r.o. and contributors. Use of this source code is governed by the Apache 2.0 license.
+
+package org.jetbrains.intellij.platform.gradle.tasks.aware
+
+import org.gradle.api.provider.Property
+import org.gradle.api.tasks.Internal
+import org.jetbrains.intellij.platform.gradle.Constants.Constraints
+import org.jetbrains.intellij.platform.gradle.models.ProductInfo
+import org.jetbrains.intellij.platform.gradle.models.validateSupportedVersion
+import org.jetbrains.intellij.platform.gradle.utils.toVersion
+
+/**
+ * The interface provides the possibility to run IDE in Split Mode.
+ */
+interface SplitModeAware : IntelliJPlatformVersionAware {
+
+    /**
+     * Enables Split Mode when running the IDE.
```

Comment:
We should provide an explanation what "Split Mode" means somewhere to make it visible to the user.
Also, it would be good to have it in the code as well, so I would copy the explanation from my pull request to KDoc of this class.

Reply:
Documentation will be provided in SDK Docs and code — soon.